MetaMask/metamask-extension
# fix/migrate AvatarAccount to TS

## Explanation

- Fixes #18882

## Screenshots/Screencaps

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/39fa931f-fe8e-4e71-aa8d-df251efd2a3d

### After

https://github.com/MetaMask/metamask-extension/assets/8112138/6b903ad3-4a12-415a-833d-6f38ac2eef79

![image](https://github.com/MetaMask/metamask-extension/assets/79097544/3069c5df-42ea-4d92-bb7d-10e6f7b70999)


## Manual Testing Steps

- Pull this branch
- Run `yarn storybook` to get storybook running locally
- Search `AvatarAccount` 
- Check stories and documentation as well as typescript code 

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
